bin/iam-token.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# the iam-token.sh scripts generate new IAM token
```

Comment:
Valentin, how about renaming it to `create-iam-token.sh`?

Reply:
done